### DISCUSSION TITLE
Allow git format-patch with --find-copies-harder

### DISCUSSION THREAD

**Discussion by RuijieYu** [magit/magit#4924] [Support] [answered]
Currently this is impossible.  Can we allow `--find-copies-harder` or double `-C` (according to the man page they are equivalent) in the `git-format-patch` interface?  Thanks.

**Comment by kyleam** [comment, marked as answer]
> Currently this is impossible

Here are two ways it's currently possible.

 * configure `diff.renames` to `copies` (e.g., `git config diff.renames copies`) then `-C` has the same effect as `-C -C`

 * add `--find-copies-harder` to the `magit-patch-create` transient

   For example

   ```elisp
   (transient-append-suffix 'magit-patch-create "-C"
     '("-H" "Inspect unmodified files to find copies" "--find-copies-harder"))
   ```

Note that the diff transient doesn't expose `--find-copies-harder` either.  In my view, leaving `--find-copies-harder` out of both of these transients is reasonable, but of course where to draw the line is subjective.

**Reply by RuijieYu** [reply]
Thank you for these two alternatives.
